source/stylesheets/refills/_progress-bar.scss

```diff
@@ -1,91 +1,66 @@
 .progress-bar {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
   $base-background-color: white !default;
-  $base-border-radius: 3px !default;
   $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  //////////////////////////////////////////////////////////////////////////////////
-
-  $progress-border-color: $base-border-color;
-  $progress-border: 1px solid $progress-border-color;
-  $progress-meter-border-color: $base-accent-color;
-  $progress-meter-border: 1px solid darken($progress-meter-border-color, 15%);
-  $progress-meter-color: $progress-meter-border-color;
-  $progress-background: darken($base-background-color, 5);
+  $progress-background-color: darken($base-background-color, 5);
+  $progress-height: 24px;
+  $progress-status-color: $base-accent-color;
+  $progress-width: 100%;
+  $progress-border-radius: $progress-height;
   $progress-animation-duration: 0.7s;
-  $progress-height: 30px;
-
-  background-color: $progress-background;
-  border: $progress-border;
-  box-shadow: inset 0 0 3px 0 rgba(darken($progress-background, 50%), 0.15);
-  border-radius: $base-border-radius;
-  height: $progress-height;
-  margin: 0 auto;
-  padding: 2px;
-  width: 100%;
 
-  > span.meter {
+  @mixin animated-progress {
     @include animation(progress $progress-animation-duration linear infinite);
-    @include box-sizing(border-box);
-    background-color: $progress-meter-color;
     @include background-image(linear-gradient(-45deg, rgba(255,255,255, 0.15) 25%,
                                                       transparent 25%,
                                                       transparent 50%,
                                                       rgba(255,255,255, 0.15) 50%,
                                                       rgba(255,255,255, 0.15) 75%,
                                                       transparent 75%));
-    background-size: 40px 40px;
+    background-color: $base-accent-color;
     background-repeat: repeat-x;
-    border: $progress-meter-border;
-    border-radius: $base-border-radius / 1.5;
-    display: block;
-    height: 100%;
-    width: 60%;
+    background-size: 40px 40px;
+    border-radius: $progress-border-radius;
   }
-}
 
-@-webkit-keyframes progress {
-  0% {
-    background-position: 0px 0px;
-  }
-  100% {
-    background-position: 40px 0px;
+  @include keyframes(progress) {
+    0% {
+      background-position: 0px 0px;
+    }
+    100% {
+      background-position: 40px 0px;
+    }
   }
-}
 
-@-moz-keyframes progress {
-  0% {
-    background-position: 0px 0px;
-  }
-  100% {
-    background-position: 40px 0px;
-  }
-}
+  @include appearance(none);
+  @include animated-progress;
+  background-color: $progress-background-color;
+  border-radius: $progress-border-radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, display, height, width